charts/merlin/values.yaml

```diff
@@ -193,6 +191,11 @@ config:
       Enabled: false
       Time: 60s
       Timeout: 5s
+    ModelClientKeepAlive:
```

Comment:
Even if these aren't specified, the app defaults will be used, correct? https://github.com/caraml-dev/merlin/blob/main/api/config/config.go#L308

Reply:
Not right now, since I've set the `default:"-"` tag on that field; I'll test it out without that default tag (so the defaults are set automatically) with the API server and see if it works there. If it still works, I'll remove it safely.